minio/replicationconfig.py

```diff
@@ -459,8 +459,6 @@ class ReplicationConfig:
     """Replication configuration."""
 
     def __init__(self, role, rules):
-        if not role:
```

Comment:
As the spec does not say this is optional, is it minio server specific behavior?

Reply:
The role is an AWS concept @balamurugana not applicable to us. This is our SDK as well. 

In-fact AWS SDK python works fine with MinIO. So yes it is not mandatory.